IBAnimatable/SlideAnimator.swift

```diff
@@ -5,18 +5,18 @@
 
 import UIKit
 
-public class SlideAnimator: NSObject, AnimatedTransitioning {
+open class SlideAnimator: NSObject, AnimatedTransitioning {
```

Comment:
Do we have any requirements to subclass transition animator like `SlideAnimator`, `SystemCameraIrisAnimator` etc. ? I think we can keep them `public`. @tbaranes Do you have any idea about this?